File: Easy/Python/119. Pascal's Triangle II.py
class Solution(object):
    def getRow(self, rowIndex):
        row = []

        for i in range(rowIndex + 1):
            row.append([])
            for j in range(i + 1):
                if j == 0 or i == j:
                    row[i].append(1)
                else:
                    row[i].append(row[i - 1][j - 1] + row[i - 1][j])

        return row[rowIndex]

File: Easy/JavaScript/119. Pascal's Triangle II.js
var getRow = function(rowIndex) {
    let row = [];

    for(let i = 0; i <= rowIndex; i++){
        row[i] = [];
        row[i][0] = 1;
        for(let j = i - 1; 0 < j; j--){
            row[i][j] = row[i - 1][j - 1] + row[i - 1][j];
        }
        row[i][i] = 1;
    }

    return row[rowIndex];
};
